docs/src/components/icons-list.astro

```diff
@@ -9,15 +9,33 @@ const icons = Object.keys(Icons) as (keyof typeof Icons)[];
 	{
 		icons.map((icon) => {
 			return (
-				<div class="icon-preview">
+				<button class="icon-preview" aria-label={`Copy ${icon} icon to clipboard`} id={icon}>
 					<Icon name={icon} size="1.5rem" />
-					<span>{icon}</span>
-				</div>
+					<span aria-live="polite">{icon}</span>
+				</button>
 			);
 		})
 	}
 </div>
 
+<script>
+  const icons = document.querySelectorAll('.icon-preview');
+  icons.forEach((icon) => {
+    icon.addEventListener('click', () => {
+      const iconName = icon.id;
+      const copiedValue = `<Icon name="${iconName}" size="2rem" />`;
```

Comment:
Do you think `size` is important to copy? It’s an optional prop (defaults to `1em`), so maybe we can keep this simple and just copy the component and name?

```suggestion
      const copiedValue = `<Icon name="${iconName}" />`;
```